screenpipe-app-tauri/scripts/pre_build.js

```diff
@@ -739,19 +846,19 @@ await copyDenoBinary();
 await installOllamaSidecar().catch(console.error);
 
 // --dev or --build
-const action = process.argv?.[2]
-if (action?.includes('--build' || action.includes('--dev'))) {
-	process.chdir(path.join(cwd, '..'))
-	process.env['FFMPEG_DIR'] = exports.ffmpeg
-	if (platform === 'windows') {
-		process.env['OPENBLAS_PATH'] = exports.openBlas
-		process.env['CLBlast_DIR'] = exports.clblast
-		process.env['LIBCLANG_PATH'] = exports.libClang
-		process.env['PATH'] = `${process.env['PATH']};${exports.cmake}`
-	}
-	if (platform == 'macos') {
-		process.env['WHISPER_METAL_EMBED_LIBRARY'] = 'ON'
-	}
-	await $`bun install`
-	await $`bunx tauri ${action.includes('--dev') ? 'dev' : 'build'}`
-}
\ No newline at end of file
+const action = process.argv?.[2];
+if (action?.includes("--build" || action.includes("--dev"))) {
+  process.chdir(path.join(cwd, ".."));
+  process.env["FFMPEG_DIR"] = exports.ffmpeg;
+  if (platform === "windows") {
+    process.env["OPENBLAS_PATH"] = exports.openBlas;
+    process.env["CLBlast_DIR"] = exports.clblast;
```

Comment:
curious what config do you use to have this change to double quote, 2 line etc.?

Reply:
Let me remove these changes... Prettier. I personally prefer to use backticks everywhere since it's faster